docs/textinput.md

```diff
@@ -321,6 +321,32 @@ If `true`, the keyboard disables the return key when there is no text and automa
 
 ---
 
+### `enterKeyHint`
+
+Determines what text should be shown to the return key. Has precedence over the `returnKeyType`
+
+_Cross platform_
```

Comment:
```suggestion
Determines what text should be shown to the return key. Has precedence over the `returnKeyType` prop.
```

Small wording tweak and cross platforms header removal, since by default docs are referring to both platforms by default.